tests/gordo_components/cli/test_cli.py

```diff
@@ -202,3 +203,124 @@ def test_expand_model_complains_on_missing_vars(self):
+                [
+                    "build",
+                    "--print-cv-scores",
+                    f"--cv-mode={cv_mode}",
+                    f"--model-location-file={location_file}",
+                ],
+            )
+            # Checks that the file is empty or not depending on the mode.
+            # Also checks the output contains 'explained-variance_raw-scores'
+            if should_be_equal:
+                assert os.stat(location_file).st_size != 0
+                assert "explained-variance_raw-scores=" in result.output
+            else:
+                assert os.stat(location_file).st_size == 0
+                assert "explained-variance_raw-scores=" in result.output
+
+
+@pytest.mark.parametrize(
+    "should_be_equal,cv_mode_1, cv_mode_2",
+    [
+        (True, "full_build", "cross_val_only"),
+        (False, "cross_val_only", "cross_val_only"),
+    ],
+)
+def test_build_cv_mode_cross_val_cache(
+    should_be_equal: bool, cv_mode_1: str, cv_mode_2: str
+):
+    """
+    Checks that cv_scores uses cache if runned after a full build. Loads the same model, and can 
+    print the cv_scores from them. 
+    """
+
+    runner = CliRunner()
+
+    logger.info(f"MODEL_CONFIG={json.dumps(MODEL_CONFIG)}")
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+
+        model_register_dir = tmpdir + "/reg"
+
+        with temp_env_vars(
+            MODEL_NAME="model-name",
+            OUTPUT_DIR=tmpdir,
+            DATA_CONFIG=DATA_CONFIG,
+            MODEL_CONFIG=json.dumps(MODEL_CONFIG),
+            MODEL_REGISTER_DIR=model_register_dir,
+        ):
+
+            if should_be_equal:
+                runner.invoke(cli.gordo, ["build", f"--cv-mode={cv_mode_1}"])
+                runner.invoke(cli.gordo, ["build", f"--cv-mode={cv_mode_2}"])
+                assert os.path.exists(model_register_dir)
+            else:
+                runner.invoke(cli.gordo, ["build", f"--cv-mode={cv_mode_1}"])
+                runner.invoke(cli.gordo, ["build", f"--cv-mode={cv_mode_2}"])
+                assert not os.path.exists(model_register_dir)
+
+
+def test_build_cv_mode_build_only():
+    """
+    Testing build with cv_mode set to build_only. Checks that the model-location-file exists and
+    are not empty. It also checks that the metadata contains cv-duration-sec=None and cv-scores={}
+    """
+
+    runner = CliRunner()
+
+    logger.info(f"MODEL_CONFIG={json.dumps(MODEL_CONFIG)}")
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        with temp_env_vars(
+            MODEL_NAME="model-name",
+            OUTPUT_DIR=tmpdir,
+            DATA_CONFIG=DATA_CONFIG,
+            MODEL_CONFIG=json.dumps(MODEL_CONFIG),
+        ):
+
+            location_file = f"{os.path.join(tmpdir, 'location.txt')}"
+            metadata_file = f"{os.path.join(tmpdir, 'metadata.json')}"
+            runner.invoke(
+                cli.gordo,
+                [
+                    "build",
+                    f"--cv-mode=build_only",
```

Comment:
Don't really need the string formatting here, unless you were aiming to parameterize this?